_SQLServer/TableLoadTreasure.sql

```diff
@@ -0,0 +1,408 @@
+'http://htkb.info/SDE/1.0/Cards/Treasure/Generic/Card_Loot_1.0_back.png', --? Is there no back?
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Treasure/Generic/Sword_Of_Alacrity.xhtml', 
+'Treasure', '1.0', '1.0 Core Set', 'Default',
+'Parry, Parry, Thrust, Thrust!');
+insert into Equipment (Position, Effect) VALUES ('Top', '+1R Attack, +1Star'); --? Not sure that this should be "Equipment" and the values need to be checked - everything about this line needs to be checked 
+
+
+--1.0
+--Treasure Specific(Used Treasure from DataLoad_Scratch file)
+
+
+--Assassin's Crossbow
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor) VALUES ('Assassin''s Crossbow',
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Treasure_Assassins_Crossbow_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Loot_1.0_back.png', --? Is there no back?
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Treasure/Specific/Assassins_Crossbow.xhtml', 
+'Treasure', '1.0', '1.0 Core Set', 'Default',
+'I always get my prey.');
+insert into Equipment (Position, Effect) VALUES ('Top', '+1G DEX'); --? The values need to be checked - everything about this line needs to be checked - Card had Pain on it - should this information go somewhere?
+
+
+--Bat Winged Key
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor) VALUES ('Bat Winged Key!',
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Treasure_Bat_Winged_Key_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Loot_1.0_back.png', --? Is there no back?
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Treasure/Specific/Bat_Winged_Key.xhtml', 
+'Treasure', '1.0', '1.0 Core Set', 'Default',
+'Flap, Flap, Flap, Flap, Flap, Flap, Flap');
+insert into Equipment (Position, Effect) VALUES ('Top', '+1R DEX, A model witth Luck may gain a Potion instead of a Heart when rolled on the dice and vice versa.  In addition, when opening a treasure chest they may draw two cards, from the treasure deck and equip one, discarding the other.'); --? Not sure that this should be "Equipment" and the values need to be checked - everything about this line needs to be checked 
+
+
+--Bewitched Wand
+insert into Cards (Name, PictureFront, PictureBack, Link, CardType, ProductSet, Module, Mode, Flavor) VALUES ('Bewitched Wand',
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Treasure_Bewitched_Wand_1.0.png', 
+'http://htkb.info/SDE/1.0/Cards/Treasure/Specific/Card_Loot_1.0_back.png', --? Is there no back?
+'http://htkb.online/SDWiki/SDE/1.0/Cards/Treasure/Specific/Bewitched_Wand.xhtml', 
+'Treasure', '1.0', '1.0 Core Set', 'Default',
+'Chilled to the bone.');
+insert into Equipment (Position, Effect) VALUES ('Top', '+1G WILL'); --? Not sure that this should be "Equipment" and the values need to be checked - everything about this line needs to be checked - Card had Stealth on it - should this information go somewhere?
+
```

Comment:
Keyword assignment line is missing for "Cold".
Keyword assignment line is missing for "Vulnerable: Fire".